numba/tests/test_objects.py

```diff
@@ -30,6 +29,11 @@ def delattr_usecase(o):
 
 
 class TestAttributes(TestCase):
+    def test_getattr(self, flags=enable_pyobj_flags):
+        import numba.tests.test_analysis as test_analysis
+        def f():
+            return test_analysis._GLOBAL
```

Comment:
I think the global needs to be a type like string that would need the builder for lowering.